lib/bulkhead/scopes/ClassScope.ts

```diff
@@ -0,0 +1,16 @@
+import { Bulkhead } from '../Bulkhead';
+
+export class ClassScope {
+  private readonly bulkheadInstance;
```

Comment:
Does the `Instance` suffix bring here any valuable context? I mean it's already a variable, it's already an instance.  
https://github.com/labs42io/clean-code-typescript#dont-add-unneeded-context